MAUI/SunburstChart/Overview.md

```diff
@@ -13,10 +13,7 @@ Syncfusion .NET MAUI Sunburst Chart is used to represent the hierarchical data s
 
 ## Key features
 
-* User interaction: The sunburst chart's legend and tooltips enhance the end-user experience significantly.
-
-* The end-user experience is empowering for exploring hierarchical data, comprehending relationships, and making informed decisions by including features such as levels, data labels, and tooltip interaction.
-
-* Legends provide more information about the segments. If an item exceeds the available bounds, the legends can be scrolled.
-
-* Provides various options for customizing and visualizing the sunburst chart features, title, data labels, legends, tooltip, and etc.
+* Supports data labels and legends for better readability.
+* Supports tooltip to provide more information about the segments.
+* Can customize appearance with own palette brushes.
```

Comment:
Can customize appearance with custom palette brushes.

Reply:
Updated the content sir